bdi_game/src/render/sdl.rs

```diff
@@ -0,0 +1,102 @@
+pub use super::traits::*;
+use sdl2;
+
+/// SDL unit
+pub struct UnitSDL<'a> {
+    pub position: &'a (i32, i32),
```

Comment:
[NOT OUTDATED] It's not about memory it's about having reference to existing object so if we update it's location then the rendering will also be updated. My pipeline caches objects to be rendered and it is workaround to make updates easier. But there are still issues with it:
1. What happens when original entity is destroyed? Because of explicit lifetimes all clusters using this object are destroyed -> in the case when compiler don't find issue, we have runtime error
2. Also i32? If we want to pass screen's coordinates and not game's coordinates then it's meaningless because screen can have only positive numbers in SDL -> we need to decide where translation to the correct coordinates happens